lib/notifier.js

```diff
@@ -25,18 +27,29 @@ function notifyIfChanged(businesses) {
 }
 
 function notifyConsole(message, options){
+    let date_ob = new Date();
     if(options.clear){
         console.clear();
     }
+    if(options.include_time) {
+        console.log(date_ob.getHours()+':'+date_ob.getMinutes()+':'+date_ob.getSeconds()+'\n');
+    }
     console.log(message + '\n');
 }
 
 function notifyDesktop(message){
     notifier.notify({ title: 'TooGoodToGo', message });
 }
 
-function createMessage(businesses){
+function createMessage(businesses,options){
     return businesses
-        .map(business => `${ business.display_name } - ${business.items_available}`)
-        .join('\n');
+        .filter(business => (business.items_available >= options.min_stock) )
+        .map(business => (
+			 (options.format==2)? `<b>${ business.display_name }</b> @ ${new Date(Date.parse(business.pickup_interval.start)).toLocaleTimeString('en-GB',business.store.store_time_zone).substring(0,5)}`
```

Comment:
1. Use a switch statement instead of chaining ternary operators.
2. Use descriptive strings instead of numbers.
```js
switch(options.format){
case 'minimal':
// etc
}
``